content/upcoming/enry.md

```diff
@@ -0,0 +1,183 @@
+---
+author: mcarmonaa
+date: 2017-07-03
+draft: true
+title: "enry: detecting your accent"
+image: ""
+description: "Announcing enry, a faster implementation in Go of github/linguist"
+categories: ["technical", "Go"]
+---
+
+<style>
+p.dt {
+  margin-top: -16px;
+  font-style: italic;
+}
+</style>
+
+## enry (⊙.☉)?
+If you don't know yet what is *enry* looking at the so descriptive name of "enry", maybe it's because you aren't the kind of person keen on films. Don't worry about it and keep reading.
+
+*[enry](https://github.com/src-d/enry)* is a tool written in Go to perform file language detection. It started as a port of github's project *[linguist](https://github.com/github/linguist)*.
+
+With all the tools that *enry*  provides,  you can know what is the language a file was written, whether this file is used as documentation for a project, is a vendor file or even if it is a binary file.
+
+## Why develop a port?
+Some time ago at source{d}, we needed to detect language for every file in every repository to perform analyzing job in our previous pipeline. To face this task, we looked at [github's linguist project](https://github.com/github/linguist), a Ruby library that offered what we needed, but not addressed all our requirements thought.
+
+So it raised the need to implement an approach to fit in an environment mostly developed in Go, give a good performance and keep compatibility with its mother project. That's the reason why *enry* was born.
+
+At the beginning, *enry* had only a bunch of the  functionalities *linguist* had, enough to cover those detection cases which were precised in that moment.
```

Comment:
`enough to cover the needed which was a requirement in that moment.`

better this way?